Mention video memory clearing on Tails (#2399)
privacyguides/privacyguides.org: docs/desktop.md

    [:octicons-home-16: Homepage](https://tails.boum.org/){ .md-button .md-button--primary }
    [:octicons-info-16:](https://tails.boum.org/doc/index.en.html){ .card-link title=Documentation}
    [:octicons-heart-16:](https://tails.boum.org/donate/){ .card-link title=Contribute }
    
<div class="admonition warning" markdown>
<p class="admonition-title">Warning</p>

> Tails doesn't erase the [video memory](https://en.wikipedia.org/wiki/Dual-ported_video_RAM). When you restart your computer after using Tails, it might briefly display the last screen that was displayed in Tails. If you shut down your computer instead of restarting it, the video memory will erase itself automatically after some time.
> 
> The issue is currently tracked in [GitLab](https://gitlab.tails.boum.org/tails/tails/-/issues/5356)

Reviewer: ```suggestion
Tails [doesn't erase](https://gitlab.tails.boum.org/tails/tails/-/issues/5356) the [video memory](https://en.wikipedia.org/wiki/Dual-ported_video_RAM) when shutting down. When you restart your computer after using Tails, it might briefly display the last screen that was displayed in Tails. If you shut down your computer instead of restarting it, the video memory will erase itself automatically after being unpowered for some time.
```
